common/static/common/js/spec/components/.gitignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
What's the purpose of this? Why do we want Git to ignore these files? I'm not sure that this should be checked in.